stable/zeppelin/templates/deployment.yaml

```diff
@@ -8,10 +8,10 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
-  strategy:
```

Comment:
why change the strategy (although it would be better to have it parameterized into `values.yaml`?

Reply:
I am open to parameterizing it. Rolling update is the default, so it seemed unnecessary to keep. I can put it back if explicitness is desired.